keras/layers/convolutional.py

```diff
@@ -320,13 +324,34 @@ def __init__(self, filters,
             bias_constraint=bias_constraint,
             **kwargs)
         self.input_spec = InputSpec(ndim=3)
+        self.causal = causal
+        self.check_causal_padding()
+
+    def call(self, inputs):
+        if self.causal:
+            self.check_causal_padding()
+            inputs = K.temporal_padding(inputs, (self.dilation_rate[0] * (self.kernel_size[0] - 1), 0))
```

Comment:
Nitpick, but the line would be more readable if `padding` was a list instead of a tuple (many parens here).